nikola/plugin_categories.py

```diff
@@ -315,11 +314,6 @@ def compile_string(self, data, source_path=None, is_two_file=True, post=None, la
         # This function used to have some different APIs in different places.
         raise NotImplementedError()
 
-    # TODO remove in v8
-    def compile_html(self, source, dest, is_two_file=True):
```

Comment:
The `self.compile_html(...)` call a few lines up should be replaced by `raise NotImplementedError()`.